### ISSUE TITLE
Update GitHub templates

### ISSUE THREAD

**Issue by jpmcb** [cncf/prow-github-actions#46]
The GitHub issue and bug templates could use an update. They're a bit wordy.

/assign @JDHulling 

If you'd like to take a look and give an update to the copy, go ahead!

**Comment by jpmcb** [comment]
FWIW: I really like the templates we use in [vmware-tanzu/community-edition](https://github.com/vmware-tanzu/community-edition/issues/new/choose)

**Comment by jeefy** [comment]
Closing as part of a backlog sweep after the transfer to the CNCF. There's no concrete proposal here and the original assignee is no longer involved. New maintainers may revisit templates in a later governance pass; reopen if you'd like to contribute a draft.
